templates/nginx.conf

```diff
@@ -4,9 +4,18 @@ upstream pretalx_{{ pretalx_instance_identifier }}_server {
 
 proxy_cache_path /tmp/nginx-pretalx-{{ pretalx_instance_identifier }} levels=1:2 keys_zone=pretalx_static_{{ pretalx_instance_identifier }}:10m inactive=60m max_size=250m;
 
+server {
```

Comment:
Since this is a marked change in behaviour, it should maybe be hidden behind a flag.